devlib/utils/android.py

```diff
@@ -694,12 +689,11 @@ def adb_list_devices(adb_server=None):
 
 
 def get_adb_command(device, command, adb_server=None):
-    _check_env()
     device_string = ""
     if adb_server != None:
         device_string = ' -H {}'.format(adb_server)
     device_string += ' -s {}'.format(device) if device else ''
-    return "adb{} {}".format(device_string, command)
+    return f"{adb}{device_string} {command}"
```

Comment:
Conflict to resolve here to re-add `LC_ALL`.

Reply:
This is now done in `_get_adb_parts()`